src/Query/Builder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Component\Database\Query;
+
+class Builder
```

Comment:
I feel like this class doesn't have much utility outside the context of a NamedArrayParameter. Duplication is far better than the wrong abstraction and I feel that this is the wrong abstraction. Both objects behavior depend heavily on one another.